From 5095705656e1acf8610f6f1407fa7b36a85273e0 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 28 Oct 2020 17:37:56 +0000
Subject: [PATCH 387/462] Mantis 0059194: Change BSS component to support
 connection to BSM Server

added jar declarations:
JarBimClient

JarGson
JarGsonFire
JarHamcrestCore
JarLoggingInterceptor
JarOkhttp
JarOkio
JarSwaggerAnnotations
JarThreetenbp


git-svn-id: svn://localhost/SelfServiceBuild/trunk@713 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index a12c400..0038251 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -279,6 +279,8 @@
 <property name="JarExtractBilling"            value="${JarDir}/extractBilling.jar"/>
 <property name="JarBhsServer"                 value="${JarDir}/BhsServer.jar"/>
 <property name="JarPaymentIngenicoC3"         value="${JarDir}/mips-payment-IngenicoC3.jar"/>
+<property name="JarBimClient"                 value="${JarDir}/mips-bim-client.jar"/>
+
 
 <!-- slf4j -->
 <property name="JarSlf4jApi"                value="${DIR_GLOBALS}/ThirdParty/slf4j/1.7.21/slf4j-api-1.7.21.jar"/>
@@ -309,6 +311,16 @@
 <property name="JarJacksonJrAll"            value="${DIR_GLOBALS}/ThirdParty/jackson/2.9.0/jackson-jr-all-2.9.0.jar"/>
 
 
+<!-- swagger -->
+<property name="JarGson"                    value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/gson-2.8.1.jar"/>
+<property name="JarGsonFire"                value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/gson-fire-1.8.0.jar"/>
+<property name="JarHamcrestCore"            value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/hamcrest-core-1.3.jar"/>
+<property name="JarLoggingInterceptor"      value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/logging-interceptor-2.7.5.jar"/>
+<property name="JarOkhttp"                  value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/okhttp-2.7.5.jar"/>
+<property name="JarOkio"                    value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/okio-1.6.0.jar"/>
+<property name="JarSwaggerAnnotations"      value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/swagger-annotations-1.5.18.jar"/>
+<property name="JarThreetenbp"              value="${DIR_GLOBALS}/ThirdParty/swagger/3.0.22/threetenbp-1.3.5.jar"/>
+
 <!-- jaxb -->
 <property name="JarJaxbImpl"                value="${DIR_GLOBALS}/ThirdParty/jaxb/2.2.7/jaxb-impl-2.2.7.jar"/>
 <property name="JarJaxbCore"                value="${DIR_GLOBALS}/ThirdParty/jaxb/2.2.7/jaxb-core-2.2.7.jar"/>
-- 
2.41.0.windows.1

